src/Donations/Components/DonationsForm.tsx

```diff
@@ -151,6 +151,7 @@ function DonationsForm() {
     switch (projectDetails.purpose) {
       case "funds":
         return <FundingDonations setopenCurrencyModal={setopenCurrencyModal} />;
+      case "conservation":
```

Comment:
Let's hope _conservation_ and _bouquet_ donations stay similar enough to share the same code for some time, so that using the same file here was the right decision to do.

Reply:
for now, they are similar enough to share the same code. But if needed, we can make the changes in the code without much effort.